.github/renovate.json

```
@@ -1,6 +1,10 @@
 {
 	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
 	"automerge": true,
+	"extends": [
+		"config:best-practices",
+		"replacements:all"
+	],
```

Appeasing the auto-formatter:

```suggestion
	"extends": ["config:best-practices", "replacements:all"],
```